controllers/datadogagent/utils.go

```diff
@@ -890,6 +890,12 @@ func getEnvVarsForAgent(logger logr.Logger, dda *datadoghqv1alpha1.DatadogAgent)
 				Value: strconv.FormatBool(*spec.Agent.Config.Dogstatsd.DogstatsdOriginDetection),
 			},
 		)
+		envVars = append(envVars,
```

Comment:
Can we put iti together in the previous `append()` ?

```
envVars = append(envVars,
			corev1.EnvVar{
				Name:  apicommon.DDDogstatsdOriginDetection,
				Value: strconv.FormatBool(*spec.Agent.Config.Dogstatsd.DogstatsdOriginDetection),
			},
corev1.EnvVar{
				Name:  apicommon.DDDogstatsdOriginDetectionClient,
				Value: strconv.FormatBool(*spec.Agent.Config.Dogstatsd.DogstatsdOriginDetection),
			},
		)
```

Reply:
Yup sure 👍 
```
diff --git a/controllers/datadogagent/utils.go b/controllers/datadogagent/utils.go
index 81ba396d..fe989f55 100644
--- a/controllers/datadogagent/utils.go
+++ b/controllers/datadogagent/utils.go
@@ -884,18 +884,16 @@ func getEnvVarsForAgent(logger logr.Logger, dda *datadoghqv1alpha1.DatadogAgent)
        envVars = append(envVars, commonEnvVars...)

        if isDogstatsdConfigured(&spec) {
-               envVars = append(envVars,
-                       corev1.EnvVar{
+               envVars = append(envVars, []corev1.EnvVar{
+                       {
                                Name:  apicommon.DDDogstatsdOriginDetection,
                                Value: strconv.FormatBool(*spec.Agent.Config.Dogstatsd.DogstatsdOriginDetection),
                        },
-               )
-               envVars = append(envVars,
-                       corev1.EnvVar{
+                       {
                                Name:  apicommon.DDDogstatsdOriginDetectionClient,
                                Value: strconv.FormatBool(*spec.Agent.Config.Dogstatsd.DogstatsdOriginDetection),
                        },
-               )
+               }...)
                // Always add DD_DOGSTATSD_SOCKET env var, to allow JMX-Fetch to use it inside pod's containers.
                envVars = append(envVars, getEnvVarDogstatsdSocket(dda))

```